cpp-ch/local-engine/Storages/SubstraitSource/HiveTextFormatFile.h

```diff
@@ -0,0 +1,60 @@
+#pragma once
+
+#include "config.h"
+#include <memory>
+#include <Columns/IColumn.h>
+#include <IO/ReadBuffer.h>
+#include <IO/PeekableReadBuffer.h>
+#include <Storages/SubstraitSource/FormatFile.h>
+#include <Processors/Formats/IRowInputFormat.h>
+#include <Processors/Formats/Impl/CSVRowInputFormat.h>
+
+namespace local_engine
+{
+class HiveTextFormatFile : public FormatFile
```

Comment:
Can it be renamed to `TextFormatFile`? Since FileSourceScanExecTransformer can also read csv or text format, it may have the same parser.

Reply:
done